python-threatexchange/threatexchange/hashing/pdq_faiss_matcher.py

```diff
@@ -101,6 +101,52 @@ def search(
             for i in range(len(query_vectors))
         ]
 
+    def search_with_distance(
+        self,
+        queries: t.Sequence[PDQ_HASH_TYPE],
+        threshhold: int,
```

Comment:
Why not call this variable distance, or the method "search_with_threshold" ?

Reply:
with_distance is about the return result. updating to `search_with_distance_in_result`